sklearn/multioutput.py

```diff
@@ -228,6 +228,20 @@ class MultiOutputRegressor(RegressorMixin, MultiOutputEstimator):
     ----------
     estimators_ : list of ``n_output`` estimators
         Estimators used for predictions.
+
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
You can go with a simpler example that one can easily see what the `MultiOutputRegressor` does:

```py
import numpy as np
from sklearn.datasets import make_regression
from sklearn.multioutput import MultiOutputRegressor
from sklearn.linear_model import LinearRegression

X = [[0], [1], [2]]
y_multi = [[0, 2], [3, 0], [6, -2]]
clf = MultiOutputRegressor(LinearRegression())
clf.fit(X, y_multi)
clf.predict([[0.5], [-1]])
```

Reply:
Sounds good! Will look into it.

Reply:
@adrinjalali open

@sbehren will you still be working on this?

Reply:
Sorry @Malesche I lost track of this. @nilchen took care.